tests/core/config/checkers/test_migration_config_checker.py

```diff
@@ -72,9 +64,9 @@ def test_check_valid_version(caplog):
     Config.unblock_update()
 
     with patch("sys.argv", ["prog", "--production", "2.0"]):
+        # No SystemExit should be raised
         core = Core()
         core.run()
-    assert caplog.text == ""
```

Comment:
Don't we want to test the generated logs anymore?

Reply:
Same reason as above